src/main/java/net/snowflake/ingest/utils/Constants.java

```diff
@@ -83,7 +83,14 @@ public enum BdecVersion {
     ONE(1),
 
     /** Uses Arrow to generate BDEC chunks with {@link ArrowBatchWriteMode#FILE}. */
-    TWO(2);
+    TWO(2),
+
+    /**
+     * Uses Parquet to generate BDEC chunks with {@link
+     * net.snowflake.ingest.streaming.internal.ParquetRowBuffer}. This version is experimental and
+     * WIP at the moment.
+     */
+    THREE(3);
```

Comment:
For Parquet, are you doing the compression at column level or chunk level?

Reply:
Ah, so we're doing page level instead of column or chunk level? Would be good to mention it in the comment, thanks!